docs/users_guide/admin_index.md

```diff
@@ -368,22 +368,91 @@ Click "Accept & match selected trainees to chosen training." The search screen w
 
-![AMY training progress steps](images/new_training_progress.png)
+![All trainees page](images/all_trainees.png)
 
-* **Trainee** Start typing in the person's name.  Auto-completed suggested names will appear.
-* **Type** This will be the training event (Training), the welcome session (Welcome Session), the teaching demo (Demo), or the lesson contribution (Lesson Contribution).  The lesson contribution type requires a link to the GitHub issue or PR.
+The colour of each item represents its state:
+* Green: Passed
+* Blue: Asked to repeat
+* Yellow: Not evaluated yet
+* Red: Failed
+
+Hover over any item to see more information about it, including its state, date, and any notes.
+
+Click on the plus sign next to a trainee's progress to track a new step for that trainee. Click on the "information" or "pencil" icon at the right of each row to view or edit the trainee's profile.
+
+Each person's individual record also has a line for Instructor Training Progress. 
+
+![AMY checkout progress in instructor profile](images/instructor_training_progress.png)
+
+Again, hover over any item to see more information, or click the plus sign to add another item.
+
+#### Adding checkout progress one by one
+
+Click the plus sign in the "Trainees" page or an individual person's record to add a new progress item.
+
+![AMY checkout progress steps](images/new_training_progress.png)
+
+* **Trainee** The page will pre-fill the trainee if possible. Otherwise, start typing in the person's name.  Auto-completed suggested names will appear.
+* **Type** This will be the training event (Training), the welcome session (Welcome Session), the teaching demo (Demo), or a community contribution (Get Involved). When a choice is made, additional fields may appear (see below).
 * **State** For the checkout type noted above, indicate if the trainee passed, was asked to repeat, or failed.  Failed should only be used in extreme circumstances.
-* **Notes** Any free notes from the admin.
+* **Notes** Any free notes from the admin. Notes are required if the state is set to Failed, or if certain Get Involved activities have been selected.
+
+Additional fields will appear depending on the Type selection:
+
+* **Event** The training event that the trainee attended. The trainee should have a learner task at this event (see [Assigning people to events](#assigning-people-to-events)). This field is only required when **Type** is set to Training.
+* **Get Involved activity** The activity that the trainee completed for the Get Involved step. This field is only required when **Type** is set to Get Involved.  
+Each activity, when selected, requires some or all of the **URL**, **Date**, and **Notes** fields to be filled as well. More information on these requirements can be found in the
+[community user guide](./community_index.md#submitting-your-get-involved-step). When you submit the form, the form will check that the correct fields have been filled and tell you if changes are needed. 
```

Comment:
Note this link won't work until #2509 is merged, as this section is added in that PR, but I didn't want to pull all those changes into this branch as it would make the review more tricky.